optimum/intel/neural_compressor/quantization.py

```diff
@@ -285,6 +285,7 @@ def get_calibration_dataset(
         preprocess_function: Optional[Callable] = None,
         preprocess_batch: bool = True,
         use_auth_token: bool = False,
+        cache_dir: Optional[str] = None,
```

Comment:
Please remove it from INC part and keep it only for OpenVINO.

Reply:
done